dax/main.tf

```diff
@@ -0,0 +1,102 @@
+provider "aws" {
+  region = var.region
+  default_tags {
+    tags = var.tags
+  }
+}
+
+data "aws_caller_identity" "current" {}
+resource "aws_iam_role" "role" {
+  name = "${var.stage}-${var.name}-dax"
```

Comment:
Same as https://github.com/peak-ai/terraform-modules/pull/13/files#r765448902
The user of the module should pass in the STAGE
Just think of any AWS resource, they don't force you to add stage to the resources